teradata/metadata.csv

```diff
@@ -1,45 +1,45 @@
-teradata.disk_space.curr_spool,gauge,,byte,,"Spool space per AMP currently being used by the user",0,teradata,currspool,
-teradata.disk_space.curr_persist_spool,gauge,,byte,,"Persistent spool space per AMP currently being used by the user",0,teradata,curr_pers_spool,
-teradata.disk_space.curr_temp,gauge,,byte,,"The number of bytes currently used by a temporary table per virtual processor (vproc)",0,teradata,curtemp,
-teradata.disk_space.peak_perm,gauge,,byte,,"Maximum amount of permanent space per AMP that has been used by the database since the last time the DBC.ClearPeakDisk macro was run",0,teradata,peakperm,
-teradata.disk_space.peak_spool,gauge,,byte,,"Maximum spool space, in bytes, that was used at any one time by the database on a specified AMP",0,teradata,peakspool,
-teradata.disk_space.peak_persist_spool,gauge,,byte,,"Maximum persistent spool space, in bytes, that was used at any one time by the database on a specified AMP",0,teradata,peak_pers_spool,
-teradata.disk_space.peak_temp,gauge,,byte,,"Peak number of bytes used at one time by a temporary table per vproc",0,teradata,peaktemp,
-teradata.disk_space.max_prof_spool,gauge,,byte,,"the profile SPOOL space limit per AMP for the user if the user is assigned a profile which has a SPOOL space setting",0,teradata,max_prof_spool,
-teradata.disk_space.max_prof_temp,gauge,,byte,,"the profile TEMPORARY space limit per AMP for the user if the user is assigned a profile which has a TEMPORARY space setting",0,teradata,max_prof_temp,
-teradata.disk_space.alloc_perm,gauge,,byte,,"current total allocation in bytes for permanent and user defined temporary table space",0,teradata,allocperm,
-teradata.disk_space.alloc_spool,gauge,,byte,,"current total allocation in bytes for spool and other system defined temporary table space",0,teradata,allocspool,
-teradata.disk_space.alloc_temp,gauge,,byte,,"current total allocation in bytes for temporary tables space",0,teradata,alloctemp,
-teradata.disk_space.perm_skew,gauge,,,percent,"permissible skew limit percent for permanent space usage at the AMP level",0,teradata,perkskew,
-teradata.disk_space.spool_skew,gauge,,,percent,"permissible skew limit percent for spool space usage at the AMP level",0,teradata,spoolskew,
-teradata.disk_space.temp_skew,gauge,,,percent,"permissible skew limit percent for temporary space usage at the AMP level",0,teradata,tempskew,
-teradata.amp.cpu_time,gauge,,second,,"number of seconds of AMP insert CPU time used by the user and account",0,teradata,cputime,
-teradata.amp.disk_io,gauge,,,,"total number of reads and writes to disk by each AMP in the system",0,teradata,diskio,
-teradata.amp.cpu_time_norm,gauge,,second,,"normalized AMP CPU time used by the user and account",0,teradata,cputime_norm,
+teradata.disk_space.max_perm.total,gauge,,byte,,"Maximum permanent space, in bytes, that is allocated to the database on a specified AMP (database level)",0,teradata,maxperm_tot,
```

Comment:
Does this mean they are tagged by db and table?

Reply:
The `.total` metrics are tagged by `td_database`, but not `td_table`. I added `(database level)` to these metric descriptions to indicate that. Should the note be more descriptive/explicit?

The metrics that don't have the `.total` suffix are tagged by both `td_database` and `td_table`. These metric descriptions are noted with `(database and table level)`.

Reply:
I think either way works! It might be helpful for users to say what the metrics are tagged by as well